src/components/modules/Error/Error.view.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <message :error="true">
+    {{ error }}
+  </message>
+</template>
+
+<script>
+import Message from '@/components/common/Message';
+
+export default {
+  name: 'ErrorView',
+
+  props: {
+    error: {
```

Comment:
So, I think we can just use slot, because in cases when we would use bold font inside of error it would not work